pkg/install/1-installresources.go

```diff
@@ -668,9 +651,20 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbClient := mgmtnetwork.NewLoadBalancersClient(i.env.SubscriptionID())
+			lbClient.Authorizer = i.fpAuthorizer
+			resourceGroupIDSplit := strings.Split(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, "/")
+			resourceGroupName := resourceGroupIDSplit[len(resourceGroupIDSplit)-1]
+			loadBalancer, err := lbClient.Get(ctx, resourceGroupName, "aro-internal-lb", "")
+			if err != nil {
+				return err
+			}
+			ipAddress = *((*loadBalancer.FrontendIPConfigurations)[0].PrivateIPAddress)
```

Comment:
is it work checking here if it not nil? Populated? And there is this nice package `"github.com/Azure/go-autorest/autorest/to"` which helps to convert ptr, to primitives and versus.